apps/go-mod-validator/src/github.ts

```diff
@@ -0,0 +1,114 @@
+import { Octokit } from "octokit";
+
+async function getDefaultBranch(
+  owner: string,
+  repo: string,
+  octokitClient: Octokit,
+): Promise<string> {
+  const repoObject = await octokitClient.request(
+    `GET /repos/${owner}/${repo}/`,
+  );
+  return repoObject.data.default_branch;
```

Comment:
The octokit client has endpoints as properties, you can see an example here: https://github.com/smartcontractkit/.github/blob/838bbbd24e5c675e0b48844a3fe6593f2bf5eb4a/actions/gha-workflow-validator/src/github.ts#L21

Reply:
ohh yes, that looks good. Implemented using that